resources/js/langman.js

```diff
@@ -80,9 +80,9 @@ new Vue({
          * Add a new translation key
          */
         addNewKey(key) {
-            if (this.translations[this.baseLanguage][key] !== undefined) {
+            /*if (this.translations[this.baseLanguage][key] !== undefined) {
                 return alert('This key already exists.');
-            }
+            }*/ //Really annoying behavior...
```

Comment:
Because you can't add a key multiple times, so if it already exists the alert prevents you from re-adding it.